packages/webpack/lib/configs/build.js

```diff
@@ -33,7 +34,7 @@ module.exports = function getConfig(config, configureBuild) {
           {
             modules: false,
             useBuiltIns: true,
-            targets: { browsers: config.browsers },
+            targets: { browsers: loadConfig({ path: config.rootDir }) },
```

Comment:
How about keeping backwards compatibility and write it like: `targets: { browsers: config.browsers || loadConfig(...) }`?

Reply:
Seeing that `browserslist` itself goes out of its way to prevent [conflicting configurations](https://github.com/browserslist/browserslist/blob/master/node.js#L250) from lurking, I am somewhat opposed to this.

Imagine you have your own browserslist config in your project and install some preset containing a browsers field: that preset would override your custom config...